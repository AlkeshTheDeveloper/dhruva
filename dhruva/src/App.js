import React from "react";
import Homepage from "./component/Homepage";
import "./App.css";

const App = () => {
  return (
    <>
      <Homepage />
    </>
  );
};

export default App;
