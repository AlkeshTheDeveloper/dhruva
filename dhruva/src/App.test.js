import { render, screen } from "@testing-library/react";
import App from "./App";

describe("React Demo", () => {
  beforeAll(() => {
    jest.spyOn(console, "warn").mockImplementation(() => {});
    jest.spyOn(console, "error").mockImplementation(() => {});
  });

  afterAll(() => {
    console.warn.mockRestore();
    console.error.mockRestore();
  });

  it("displays title in page", () => {
    render(<App />); // Render the component
    const title = screen.getByText("React Demo"); // Find the title element
    expect(title).toMatchSnapshot(); // Snapshot the specific element
  });
});
