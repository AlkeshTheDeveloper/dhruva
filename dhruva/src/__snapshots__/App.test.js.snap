// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React Demo displays title in page 1`] = `
<h1>
  React Demo
</h1>
`;
